signalr-node-server/signalr.js

```diff
@@ -353,9 +355,8 @@ class HubContext {
 }
 
 class Hub {
-    _methods = new Map();
-
     constructor() {
+        this._methods = new Map();
```

Comment:
Why did you make this a member?

Reply:
This was the part where Node v10 failed. I figured this would be an easy solution but not sure